plugins/modules/route53_health_check.py

```diff
@@ -121,12 +131,16 @@
     import boto.ec2
     from boto.route53 import Route53Connection, exception
     from boto.route53.healthcheck import HealthCheck
+    # Boto 2 is mandatory, Boto3 is only needed for tagging
+    import botocore
 except ImportError:
-    pass  # Handled by HAS_BOTO
+    pass  # Handled by ec2.HAS_BOTO and ec2.HAS_BOTO3
 
 from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
 from ansible_collections.amazon.aws.plugins.module_utils.ec2 import HAS_BOTO
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import HAS_BOTO3
 from ansible_collections.amazon.aws.plugins.module_utils.ec2 import get_aws_connection_info
+from ansible_collections.community.aws.plugins.module_utils.route53_tags import manage_tags
```

Comment:
```suggestion
from ansible_collections.community.aws.plugins.module_utils.route53 import manage_tags
```
I'd rather we name the module_utils file as just `route53.py` so it can readily/obviously be used for other utilities if any are created in the future.